src/main/java/com/taxjar/Taxjar.java

```diff
@@ -96,6 +98,18 @@ public void buildClient(Map<String, Object> params) {
         apiService = retrofit.create(Endpoints.class);
     }
 
+    private static String getUserAgent() {
+        String[] propertyNames = {"os.name", "os.version", "os.arch", "java.version", "java.vendor"};
+        Set<String> properties = new LinkedHashSet<String>();
+
+        for (String property : propertyNames)
```

Comment:
I would recommend[ always using the braces ](https://stackoverflow.com/a/8020255/783284)in `for` loops, even though it is technically allowed.